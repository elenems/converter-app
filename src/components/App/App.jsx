import React from "react";
import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
import HomePage from "../../pages/HomePage";
import PageNotFound from "../../pages/NotFoundPage";

function App() {
  return (
    <Router>
      <div className="App">
        <Switch>
          <Route path="/" component={HomePage} exact />
          <Route component={PageNotFound} />
        </Switch>
      </div>
    </Router>
  );
}

export default App;
