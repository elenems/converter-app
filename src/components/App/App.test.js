import App from "../App";
import React from "react";
import { shallow } from "enzyme";

it("Renders App", () => {
  const wrapper = shallow(<App />).debug();
  expect(wrapper).toMatchSnapshot();
});
