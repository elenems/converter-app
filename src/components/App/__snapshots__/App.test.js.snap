// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App 1`] = `
"<BrowserRouter>
  <div className=\\"App\\">
    <Switch>
      <Route path=\\"/\\" component={[Function: HomePage]} exact={true} />
      <Route component={[Function: PageNotFound]} />
    </Switch>
  </div>
</BrowserRouter>"
`;
